pkg/api/norman/customization/aks/listers.go

```diff
@@ -158,7 +158,11 @@ func listKubernetesUpgradeVersions(ctx context.Context, clusterLister mgmtv3.Clu
 		return nil, http.StatusBadRequest, fmt.Errorf("invalid cluster id")
 	}
 
-	resp.CurrentVersion = *cluster.Spec.AKSConfig.KubernetesVersion
+	if cluster.Spec.AKSConfig.KubernetesVersion != nil {
+		resp.CurrentVersion = *cluster.Spec.AKSConfig.KubernetesVersion
+	} else {
+		resp.CurrentVersion = *cluster.Status.AKSStatus.UpstreamSpec.KubernetesVersion
```

Comment:
Is it possible for `*cluster.Status.AKSStatus.UpstreamSpec.KubernetesVersion` to be nil ? 
If it is not, the fix looks good to me.

Reply:
@thedadams I don't think so right? Will the kubernetes version in the status always be set?